src/core/aws/aws-project.ts

```diff
@@ -313,14 +316,12 @@ export default class AWSProject extends BaseProject {
             if (varFile) {
                 command += ` -var-file=${varFile}`;
             }
-            execSync(command, {
-                cwd: `${projectPath}`,
-                stdio: 'inherit',
-                env: process.env
-            });
-            AppLogger.info('Terraform destroy completed successfully.', true);
+    
+            const result = executeCommandWithRetry(command, {cwd: `${projectPath}`, stdio: 'inherit', env: "process.env" })
```

Comment:
what will be value of result? Also, we are not checking or doing anything with result..

Reply:
YEs, we are taking that out